UPGRADE.md

```diff
@@ -1,5 +1,15 @@
 # Upgrade
 
+## dev-master
+
+### Filter
+
+Update the schema `app/console doctrine:schema:update --force` and run following SQL-Statement:
+ 
+```sql
+UPDATE re_conditions SET value = CONCAT('"', value, '"') WHERE value NOT LIKE '"%"'
```

Comment:
we havent used the migrations for doctrine until now and i dont want to introduce it in master to be honest